hr-time/unload_manual.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>time origin value manual test</title>
+<link rel="help" href="https://w3c.github.io/hr-time/#time-origin-1">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+var t = async_test('Window time origin is almost equal (<10ms) to the time of the user confirming the navigation if a confirmation dialog is displayed');
+
+var child_unload_time = child_origin = 0;
+var child_load_index = 0;
+
+onmessage = t.step_func(function(event) {
+  if (event.data != "hr_time_unload_done") {
+    return;
+  } else {
+  document.getElementById('time').innerHTML = "<hr/><p>child_origin: " + child_origin + "</p><p>unload time was : " + child_unload_time + "</p><hr/>";
+
+  assert_less_than(Math.abs(child_unload_time - child_origin), 10, "");
+  
+  t.done();
+  }
+});
+</script>
+</head>
+<body>
+<h2>Description</h2>
+<p>This test validates that time origin is almost equal (less than 10ms apart) to the time of the user confirming the navigation if a confirmation dialog is displayed during the prompt to unload algorithm.</p>
+<hr/>
+
+<h2>Manual Test Steps:</h2>
+<ol>
+<li>Click the "open a secondary window" link below;</li>
+<li>Please click the "newNavigation" button to reload the page in the secondary page;</li>
+<li>Confirm the unload dialog within 10ms;</li>
```

Comment:
@spanicker sorry, this is should be 10s, the [default timeout](http://web-platform-tests.org/writing-tests/testharness-api.html#harness-timeout) of testharness.